scripts/jenkins/qa_devstack.sh

```diff
@@ -84,6 +84,12 @@ function h_setup_base_repos {
             $zypper ar -f "http://smt-internal.opensuse.org/repo/\$RCE/SUSE/Products/SLE-SERVER/${DIST_VERSION}/x86_64/product" Base || true
             $zypper ar -f "http://smt-internal.opensuse.org/repo/\$RCE/SUSE/Products/SLE-SDK/${DIST_VERSION}/x86_64/product" SDK || true
             $zypper ar -f "http://smt-internal.opensuse.org/repo/\$RCE/SUSE/Updates/SLE-SDK/${DIST_VERSION}/x86_64/update/" SDK-Update || true
+            $zypper ar -f "http://download.suse.de/ibs/SUSE:/Factory:/Head/standard" SUSE_Factory_Head || true
+        fi
+        if [[ $DIST_VERSION == 12-SP3 ]]; then
+            $zypper ar -f "https://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_12_SP3/" devel_languages_python_backports  || true
+        elif [[ $DIST_VERSION == 12-SP4 ]]; then
+            $zypper ar -f "https://download.opensuse.org/repositories/devel:/languages:/python/SLE_12_SP4/" devel_languages_python  || true
```

Comment:
Please always use dlp:backports, also for sp4

Reply:
OK.  So why is SP4 one of the d:l:p repos?

Reply:
@aspiers there is no good reason, but I'm tired fighting with the dlp maintainers about this. whenever I delete it, they add it back.